source/features/highlight-collaborators-and-own-discussions.tsx

```diff
@@ -1,21 +1,32 @@
 import './highlight-collaborators-and-own-discussions.css';
+import cache from 'webext-storage-cache';
 import select from 'select-dom';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 import fetchDom from '../helpers/fetch-dom';
 import {getRepoURL, getUsername} from '../github-helpers';
 
+const getCollaborators = cache.function(async (): Promise<string[]> => {
+	const dom = await fetchDom(getRepoURL() + '/issues/show_menu_content?partial=issues/filters/authors_content');
+	const collaborators = select.all<HTMLImageElement>('.SelectMenu-item [alt]', dom).map(collaborator => {
+		return collaborator.alt.slice(1);
+	});
+
+	return collaborators;
+}, {
+	maxAge: 5,
+	staleWhileRevalidate: 20,
+	cacheKey: () => `${getRepoURL()}:repo-collaborators`
```

Comment:
```suggestion
	cacheKey: () => 'repo-collaborators:' + getRepoURL()
```

At some point it would be good to implement [cacheKey as string](https://github.com/fregante/webext-storage-cache/issues/17) to simplify the code